.travis.yml

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I have CircleCI doing linux builds, you may not need travis. If that's the case, let's remove this.